### DISCUSSION TITLE
Power Source

### DISCUSSION THREAD

**Discussion by Kev994** [Zer0-bit/gaggiuino#231] [Q&A] [answered]
I’m not sure I understand the power section. Is option 1 to power the system via the USB as is done in the testing phase?
The 12v/1A power source is on backorder until end Aug so I’m wondering if I can use the USB in the meantime. 
also: amazing project! Thanks!

**Comment by Zer0-bit** [comment, marked as answer]
![Screenshot_20220626-082347_1](https://user-images.githubusercontent.com/42692077/175804048-72e51aee-90d1-4b0b-a773-bcac921804ac.png)
